pyproject.toml

```diff
@@ -0,0 +1,271 @@
+[tool.poetry]
+name = "pec"
+version = "0.1.0"
+description = ""
+authors = "Krzysztof Wisniewski", "Mateusz Szpyruk", "Mateusz Zalewski", "Paweł Olszewski"
+license = "MIT"
+readme = "README.md"
+homepage = ""
+repository = "https://github.com/UniversityOfGdanskTeamPython/pec_framework_data_consumer/"
+documentation = ""
+keywords = []
+classifiers = [
+    "Development Status :: 1 - Planning",
+    "Environment :: Web Environment",
+    "Intended Audience :: Education",
+    "Intended Audience :: End Users/Desktop",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
```

Comment:
We will not support Python 3.13 due to [llvmlite](https://pypi.org/project/llvmlite/) limitations.